bin/regression-wally

```diff
@@ -417,17 +426,18 @@ elif (args.fcov):  # only run RV64GC tests on Questa in lockstep in functional c
 
 else: 
     for sim in sims:
-        if (not (args.buildroot and sim == defaultsim)):  # skip short buildroot sim if running long one
+        if (not (args.buildroot and sim == lockstepsim)):  # skip short buildroot sim if running long one
             addTests(tests_buildrootshort, sim)
         addTests(tests, sim)
         addTests(tests64gc_nofp, sim)
         addTests(tests64gc_fp, sim)
 
 # run derivative configurations and lockstep tests in nightly regression
 if (args.nightly):
-    addLockstepTestsByDir(WALLY+"/tests/coverage", "rv64gc", "questa", 0)
-    addLockstepTestsByDir(WALLY+"/tests/riscof/work/wally-riscv-arch-test/rv64i_m", "rv64gc", "questa", 0)
+    addLockstepTestsByDir(WALLY+"/tests/coverage", "rv64gc", lockstepsim, 0)
+    addLockstepTestsByDir(WALLY+"/tests/riscof/work/wally-riscv-arch-test/rv64i_m", "rv64gc", lockstepsim, 0)
     addTests(derivconfigtests, defaultsim)
+    addTests(bpredtests, defaultsim)
```

Comment:
As is, adding the branch predictor tests breaks regression. Several things have changed about how `wsim` works since those were written, so regression will need to be modified if we want those to work. Probably makes sense to add this in a later PR so we can get the rest of this in.

Reply:
Sounds good, I'll comment it out with that tag